gk-up.sh

```diff
@@ -5,63 +5,119 @@
 # script creates a file named "/root/__SUCCESS" once it has completed. If the gce cluster is running it
 # will be shut down (this script is idempotent.) A custom cns s3 service broker is also installed.
 #
-# Args:
-#		-y to bypass hitting a key to acknowlege seeing the displayed config info.
-#       The default is to prompt the user to continue.
-#		-i Deploy GCE instances and stop.  Startup script will still install packages and dependencies on all VMs.
-#          DO NOT deploy k8s, gluster-kubernetes (including S3 component), or the CNS Objet Broker
-#		-g Deploy gluster-kubernetes and stop.
-#          DO NOT install S3 components or the CNS Object Broker
-#		-s Deploy gluster-kubernetes, S3 components and stop.
-#          DO NOT install CNS Object Broker
+# Args: see the help text for all of the command line arguments.
+#
 # Note: lowercase vars are local to functions. Uppercase vars are global.
 
 set -euo pipefail
 
 # Print help
 function print_help() {
-	echo "Gluster-Kubernetes GCE deployment script."
-	echo "-y to bypass hitting a key to acknowlege seeing the displayed config info."
-	echo "   The default is to prompt the user to continue."
-	echo "-i Deploy GCE instances and stop.  Startup script will still install packages and dependencies on all VMs."
-	echo "   DO NOT deploy k8s, gluster-kubernetes (including S3 component), or the CNS Objet Broker"
-	echo "-g Deploy gluster-kubernetes and stop."
-	echo "   DO NOT install S3 components or the CNS Object Broker"
-	echo "-s Deploy gluster-kubernetes, S3 components and stop."
-	echo "   DO NOT install CNS Object Broker"
+	cat <<END_HELP
+
+Gluster-Kubernetes GCE deployment script:
+
+   -y			auto answer all prompts with their default value.
+   --kube		(default) install and deploy kubernetes.
+   --no-kube		do not install nor deploy kubernetes, just install a basic GCE cluster.
+   --gluster		(default) install and deploy gluster.
```

Comment:
Do we still want defaults to be a CNS + S3 cluster?

Reply:
Also, since we're also installing non-containerized gluster for the current project, it would be nice to distinguish the 2 in the options.

Reply:
Re defaults: I was thinking the defaults should be INIT_KUBE=yes, INSTALL_GLUSTER=yes but we don't, by default, install gluster-s3 nor s-c. This is easy to change but if we agree to these new defaults then some of our doc on gluster-s3/service-catalog will need to change -- so maybe not worth it?

Reply:
The way I was thinking of it, the gk-cluster-deploy repo would remain the automated deployment method for the cns-object-broker.  That allows us to mutate this repo as we need to so that it tracks w/ whatever new project we're working on.  It was just useful to copy gk-cluster-deploy to give us a baseline.

re: installing containerized gluster as a default - Since the scope of this repo should wrap up a bunch of different components (CNS, plain gluster packages, service-catalog, brokers), should the default be just bare VMs?  And we toggle on each component we want?  (and dependent components install automatically e.g. if the broker is selected, also install kubernetes)

Reply:
I like that idea. I'll change the defaults to be only INIT_KUBE = yes (1). All other actions need to be explicitly supplied.